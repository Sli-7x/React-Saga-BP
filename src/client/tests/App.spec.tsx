import * as React from 'react';
import * as renderer from 'react-test-renderer';
import App from '../App';
import { StaticRouter } from 'react-router';

describe('App.tsx', () => {
  it('should render correctly', () => {
    const filledTree = renderer
      .create(
        <StaticRouter location="someLocation" context={{}}>
          <App />
        </StaticRouter>,
      )
      .toJSON();
    expect(filledTree).toMatchSnapshot();
  });
});
