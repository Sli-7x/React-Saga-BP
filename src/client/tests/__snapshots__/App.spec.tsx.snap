// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.tsx should render correctly 1`] = `
<div
  id="wrapper"
>
  <div
    id="header"
  >
    <a
      href="/"
      onClick={[Function]}
    >
       Home 
    </a>
    <a
      href="/about"
      onClick={[Function]}
    >
       About 
    </a>
    <a
      href="/product"
      onClick={[Function]}
    >
       Product 
    </a>
  </div>
  <main
    className="content"
  />
  <div
    id="footer"
  >
    <span
      className="sc-bdVaJa rOCEJ"
    >
      Â© 
      2018
    </span>
  </div>
</div>
`;
